distribution/ddf-common/src/main/resources/etc/default.policy

```diff
@@ -0,0 +1,120 @@
+priority "grant";
+// todo symlinks
+
+deny {
+    // Read, write, or execute any file
+    permission java.io.FilePermission "<<ALL FILES>>", "read, write, execute";
```

Comment:
Should also be blocking delete?

Reply:
It should, and future iterations (likely beyond 2.11.0) will. Could also consider blocking `readLink`, but that's of lower concern.